src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let NEXT_STATE = { ...state };
+  const RESULT = [];
```

Comment:
you don't really need uppercase here:
```suggestion
  let stateCopy = { ...state };
  const result = [];
```